libs/utils/utils.js

```diff
@@ -871,8 +871,11 @@ export async function loadArea(area = document) {
     window.dispatchEvent(new Event('milo:LCP:loaded'));
 
     // Post LCP operations.
-    if (isDoc && section.el.dataset.idx === '0') { loadPostLCP(config); }
-
+    if (window.PostLCPevnt && isDoc && section.el.dataset.idx === '0') {
+      window.addEventListener(config.PostLCPevnt, () => {
```

Comment:
Might be worth checking whether `config.PostLCPevnt` exists before adding the event listener, maybe not all projects declare it. Also, properties from `config` are usually lowercase, maybe something like `customPostLCPEvent` or similar might be better. There is already a `milo:LCP:loaded` event, that's why it might make sense to make the difference between the Milo one and a consumer one.

Reply:
I don't think we need a config property here. I think we can just use a single value `window.customPostLCPEvent` to check and listen for the event.

Reply:
Why not in config?  Everything else the consumer declares is in that object, so seems non-standard to set a global window property.  I'd rather we only use `config.postLCPevent`.

nit: no need to save one char with `evnt`, just use full `event`